File: src/net/auoeke/limitless/config/enchantment/entry/EnchantmentParticleConfiguration.java
package net.auoeke.limitless.config.enchantment.entry;

import net.auoeke.limitless.config.enchantment.entry.radius.Radius;
import me.shedaniel.autoconfig.annotation.ConfigEntry.Gui.CollapsibleObject;

public class EnchantmentParticleConfiguration {
    public boolean enabled = true;
    public boolean inherit = true;

    @CollapsibleObject
    public Radius radius = new Radius();
}

File: src/net/auoeke/limitless/config/enchantment/entry/EnchantmentParticleConfiguration.kt
package net.auoeke.limitless.config.enchantment.entry

import me.shedaniel.autoconfig.annotation.ConfigEntry.Gui.CollapsibleObject
import net.auoeke.limitless.config.enchantment.entry.radius.Radius

class EnchantmentParticleConfiguration {
    var enabled: Boolean = true
    var inherit: Boolean = true

    @CollapsibleObject
    var radius = Radius()
}
